src/components/Searchbar.stories.js

```diff
@@ -0,0 +1,11 @@
+import Searchbar from './Searchbar.js';
+
+export default {
+  title: 'Component/Searchbar',
+  component: Searchbar,
+};
+
+const Template = args => <Searchbar {...args} />;
+
+export const SearchbarAB = Template.bind({});
+SearchbarAB.args = {};
```

Comment:
Is there a special reason why you name it SearchbarAB ?

Reply:
Its actually from my filegenerator a setting - I really need to change it 🙈😅